lib/generators.nix

```diff
@@ -177,13 +177,14 @@ rec {
 
   # PLIST handling
   toPlist = {}: v: let
-    expr = ind: x: with builtins;
-      if isNull x then "" else
-      if isBool x then bool ind x else
-      if isInt x then int ind x else
+    expr = ind: x:  with builtins;
+      if isNull x   then "" else
+      if isBool x   then bool ind x else
+      if isInt x    then int ind x else
       if isString x then str ind x else
-      if isList x then list ind x else
-      if isAttrs x then attrs ind x else
+      if isList x   then list ind x else
+      if isAttrs x  then attrs ind x else
+      if isFloat x  then float ind x else
```

Comment:
`builtins.isFloat` was not available in 1.11 and we should keep nixpkgs compatible until 18.09. But we can add a fallback as long as nothing in nixpkgs itself uses floats.

eg.

```nix
let
  toFloat = builtins.toFloat or (_: false);
  ...
```